scripts/processing/01-run_anatomy.ipy

```diff
@@ -2,12 +2,12 @@
 Run Freesurfer/MNE anatomical pipeline
 ======================================
 
-This runs Freesurfer recon-all on all subjects
-and computes the BEM surfaces later used for forward
-modeling. BEM extraction is done using flash MRI
-data.
-"""
+This runs Freesurfer recon-all on all subjects and computes the BEM surfaces
+later used for forward modeling. BEM extraction is done using flash MRI data.
 
+Make sure to run `source SetUpFreeSurfer.sh` (bash and derivatives) or `source
+SetUpFreeSurfer.csh` (csh and derivatives) before running this script.
```

Comment:
It is. I'm running this on a university workstation which supposedly has Freesurfer set-up. So, I never did any of the installation. I imagine many others being in the same boat. Maybe we can re-phrase this to "read the installation section of the Freesurfer manual" or something to that regard.

What most tripped me up was that there was no clear error message. I haven't dug into it too deeply, but my guess is that the errors are printed to stderr and therefore do not end up in the logfile (which is written to by redirecting stdout). When I ran `recon-all` manually, the error `nu_correct not found` was printed to the terminal. The error does not prevent `recon-all` to merrily proceed, and only fail when it cannot find the T1 images. 

All in all it took a while for me to realize I should've called the `SetUpFreeSurfer.sh` script. Obvious in hindsight, but not so obvious from the post-mortem on the logfiles.